docs/DeveloperGuide.md

```diff
@@ -659,6 +659,53 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Modifying weight of a client
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to see a list of all clients
+1. FitBook displays a success message followed by the list of clients
+1. User views the list and enters the `weight` command to modify the weight of a client
```

Comment:
Doesn't the weight command add a new weight instead? If modifying, i think it should be the `edit` command